contracts/ProtocolReserve/RiskFundV2.sol

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { IERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import { AccessControlledV8 } from "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+import { SafeERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+
+import { IRiskFund } from "../Interfaces/IRiskFund.sol";
+import { IRiskFundConverter } from "../Interfaces/IRiskFundConverter.sol";
+
+import { ensureNonzeroAddress, ensureNonzeroValue } from "../Utils/Validators.sol";
+import { EXP_SCALE } from "../Utils/Constants.sol";
+
+import { RiskFundV2Storage } from "./RiskFundStorage.sol";
+
+/// @title RiskFundV2
+/// @author Venus
+/// @notice Contract with basic features to hold base asset for different Comptrollers
+/// @dev This contract does not support BNB
+/// @custom:security-contact https://github.com/VenusProtocol/protocol-reserve#discussion
+contract RiskFundV2 is AccessControlledV8, RiskFundV2Storage, IRiskFund {
```

Comment:
RiskFundV2 will be an upgrade of the current [RiskFund](https://bscscan.com/address/0xdF31a28D68A2AB381D42b380649Ead7ae2A76E42). The [current RiskFund implementation](https://bscscan.com/address/0x0e8ef0ec1e0c109c5b5249ccefb703a414835eac#code) is declared this way:

```
contract RiskFund is AccessControlledV8, ExponentialNoError, ReserveHelpers, MaxLoopsLimitHelper, IRiskFund
```

I think the RiskFundV2 is following the same order of declaration in the inheritance, to avoid any collision. Variables of `AccessControlledV8` will be at the beginning of the storage layout, and then the variables of the ReserveHelper (`ReserveHelpersStorage`), `MaxLoopsLimitHelpersStorage`, the variables defined in the RiskFund contract itself, and finally the new variable `riskFundConverter`.

Reply:
Yeah, that's why order of inheritance is kept in this way.
As you can see the last two images are from the current order of inheritance used, if we change order then storage layout is changing.
![Screenshot from 2023-11-30 09-33-31](https://github.com/VenusProtocol/protocol-reserve/assets/104391977/e8012d79-84f6-498b-b4c5-b1b2b4ff6850)
![Screenshot from 2023-11-30 09-33-08](https://github.com/VenusProtocol/protocol-reserve/assets/104391977/500d24b2-ce16-47c9-8d9b-bacda355546a)
![Screenshot from 2023-11-30 09-31-17](https://github.com/VenusProtocol/protocol-reserve/assets/104391977/660ddaee-6f07-44a6-a700-b9cfb63fa008)

Reply:
Thank you, LGTM